adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/_ui/responsive/common/js/adyen.checkout.js

```diff
@@ -143,7 +141,7 @@ var AdyenCheckoutHybris = (function () {
                     }
                 },
                 type: card.type,
-                hideCVC: hideCVC
+                hideCVC: false
```

Comment:
Do you also handle recurring payments on this checkout? Because then the hideCVC should still be a variable and check if it is oneclick or recurring. (In Magento we do it like this on the same handler js)

If no recurring just oneClick, then is it necessary to add this property with false or can just be avoided?

Reply:
We don't have recurring payments in hybris hence removing hideCVC flag.